packages/ui/src/TableCell/index.tsx

```diff
@@ -0,0 +1,126 @@
+import cn from 'clsx';
+import { ReactNode } from 'react';
+import { Density, useDensity } from '../utils/density';
+import { Skeleton } from '../Skeleton';
+import {
+  tableHeading,
+  tableHeadingMedium,
+  tableHeadingSmall,
+  tableItem,
+  tableItemMedium,
+  tableItemSmall,
+} from '../utils/typography';
+
+export type TableCellVariant = 'title' | 'heading' | 'cell' | 'lastCell' | 'footer';
```

Comment:
What is the difference between title and heading?

Reply:
copied from Figma but never seen `title` being actually used in there